pkg/cmd/server/apis/config/v1/types.go

```diff
@@ -711,6 +714,17 @@ type DNSConfig struct {
 	AllowRecursiveQueries bool `json:"allowRecursiveQueries"`
 }
 
+// WebhookTokenAuthenticators holds the necessary configuation options for
+// external token authenticators
+type WebhookTokenAuthenticator struct {
+	// WebhookTokenAuthnConfigFile is a path to a Kubeconfig file with the webhook configuration
+	WebhookTokenAuthnConfigFile string `json:"webhookTokenAuthnConfigFile"`
+	// WebhookTokenAuthnCacheTTL indicates how long an authentication result should be cached.
+	// It takes a valid time duration string (e.g. "5m").
+	// If empty, you get the default timeout. If zero (e.g. "0m"), caching is disabled
```

Comment:
Probably should state what the default timeout is (I see no code that does defaulting in this PR).

Reply:
There is currently no default, you are required tio state the CacheTTL

Reply:
Upstream defaults this to 2 minutes.  @liggitt any preference?

Reply:
I've added a 10s default